fdbserver/workloads/GetMappedRange.actor.cpp

```diff
@@ -413,6 +447,17 @@ struct GetMappedRangeWorkload : ApiWorkload {
 		}
 	}
 
+	ACTOR static Future<Void> testLoop(Database cx, GetMappedRangeWorkload* self, Key mapper) {
+		loop choose {
+			when(wait(delay(0.1))) {
```

Comment:
0.1 is too frequent for status. It's better to do this every 2~5 seconds.

And, putting a `wait` inside the `when` block is bad style. It's better to define a state variable and assign `reportMetric(self, cx)` to the variable.